web/src/narrow.js

```diff
@@ -367,7 +361,7 @@ export function activate(raw_operators, opts) {
         $(".tooltip").hide();
 
         blueslip.debug("Narrowed", {
-            operators: operators.map((e) => e.operator),
+            terms: terms.map((e) => e.operator),
```

Comment:
The object key should still be called `operators` actually.

Reply:
ah great catch thanks